project3/index.html

```diff
@@ -0,0 +1,251 @@
+                fill="#fafafa" /></svg>
+
+        <h1>super hero registry</h1>
+
+        <p>With great power there must also come -- great responsibility</p>
+
+    </header>
+
+
+    <form action="">
+
+        <main>
+
+            <div class="identity">
+                <h2>identity</h2>
+
+
+                <label for="real-name">Real Name</label>
+                <input id="real-name" type="text" placeholder="" required>
+
+                <label for="email">Alias</label>
+                <input id="email" type="alias" placeholder="" required>
+
+                <label for="email">Email</label>
+                <input id="email" type="email" placeholder="" required>
+
+            </div>
+
+
+
+            <div class="personal-characteristics">
+                <h2>personal characteristics</h2>
+            </div>
+
+            <div>
+                <label for="age">Age</label>
+                <input id="age" type="number" placeholder="" required>
+            </div>
+
+            <div class="category">
+                <fieldset class="category">
+                    <legend>Category</legend>
+
+                    <p>Please select as many as apply to your origin and how you gainded your powers.</p>
+
+                    <label for="alien"><input name="category" id="alien" type="checkbox">Alien</label>
+
+                    <label for="cyborg"><input name="category" id="cyborg" type="checkbox">Cyborg</label>
+
+                    <label for="cyborg"><input name="category" id="experiment" type="checkbox">Experiment</label>
+
+                    <label for="cyborg"><input name="category" id="human" type="checkbox">Human</label>
+
+                    <label for="cyborg"><input name="category" id="inhuman" type="checkbox">Inhuman</label>
+
+                    <label for="cyborg"><input name="category" id="mutant" type="checkbox">Mutant</label>
+
+                </fieldset>
+            </div>
+
+
+            <div class="gender">
+                <fieldset class="gender">
+                    <legend>Gender</legend>
+
+                    <label for="woman"><input name="gender" id="woman" type="radio">Woman</label>
+
+                    <label for="man"><input name="gender" id="man" type="radio">Man</label>
+
+
+                    <label for="nonbinary"><input name="gender" id="nonbinary" type="radio">Nonbinary</label>
+
+
+                    <label for="nonbinary"><input name="gender" id="genderqueer-genderfluid"
+                            type="radio">Genderqueer/Genderfluid</label>
+
+
+                    <label for="nonbinary"><input name="gender" id="prefer-not-to-answer" type="radio">Prefer not to
+                        answer</label>
+
+
+                    <label for="nonbinary"><input name="gender" id="prefer-to-self-describe" type="radio">Prefer to self
+                        describe</label>
+                    <textarea name="" id="prefer-to-self-describe"></textarea>
```

Comment:
This `textarea` can't have the same `id` as the `input` above; `id`s must be unique.